fast_llm/data/preprocessing/image_patch.py

```diff
@@ -58,6 +59,21 @@ class ImagePatchConfig(PreprocessingConfig):
         hint=FieldHint.optional,
     )
 
+    @classmethod
```

Comment:
Why do we need this? Was there some datasets prepared with the field?

Reply:
Yes, many image datasets have been prepared with this field. So we currently need this to read these datasets.